packages/tools/component-generator/src/index.ts

```diff
@@ -88,10 +88,11 @@ export const Default = () => <nge-${kebabCase} />;
 
 const getRegisterTemplate = (
     componentName: string,
-    kebabCase: string
+    kebabCase: string,
+    componentType: string
 ): string => {
     return `import register from 'preact-custom-element';
-import ${componentName} from './${componentName}';
+import ${componentName} from '@/${componentType}/${componentName}/${componentName}';
```

Comment:
lets keep in mind that this won't work properly until we figure out aliased imports in nge